package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "nest": "nest",
     "prepare": "husky install",
+    "dedupe": "npx yarn-deduplicate yarn.lock",
```

Comment:
Updating the SDK introduced the warning message that happen all the time.

According to some SO post, yarn 2 is more aggressive in dedupe and will likely eliminate them for us.

For now, this package does the dedupe, but lets us stay on yarn 1 for now.

It only needs to be ran when you see the warnings after updating something touching polkadot dependencies.